types/config.go

```diff
@@ -31,6 +35,7 @@ var (
 		coinType:           CoinType,
 		fullFundraiserPath: FullFundraiserPath,
 		txEncoder:          nil,
+		keyringServiceName: DefaultKeyringServiceName,
```

Comment:
can you move the `sdkConfig` var to `GetConfig()` ?

Reply:
Done

Reply:
Mmm no, the singleton is used across the whole SDK. I'll revert my previous commit